maintenance/wikia/cronjobs/AutoFollow.php

```diff
@@ -46,11 +57,31 @@
 		continue;
 	}
 
+	foreach ( $textsToWatch[$contentLanguage] as $text ) {
+		$titlesToWatch[] = Title::newFromText( $text );
+	}
+
+	$status = false;
 	foreach ( $titlesToWatch as $title ) {
-		echo "Watching {$title->getText()} as {$user->getName()}...\n";
-		WatchAction::doWatch( $title, $user );
+		$logParams = [
+			'user_id' => $row['user_id'],
+			'user_name' => $user->getName(),
+			'user_lang' => $userLanguage,
+			'title' => $title->getText(),
+		];
+		if ( $title instanceof Title ) {
+			WatchAction::doWatch( $title, $user );
+			$status = true;
+
+			\Wikia\Logger\WikiaLogger::instance()->info( "AutoFollow: User {$user->getName()} added to watchlist of {$title->getText()}.", $logParams );
```

Comment:
Same here, this would give "AutoFollow: User Foo added to watchlist of Wikia Staff Blog.", rather than "AutoFollow: User Foo added to watchlist of Blog:Wikia Staff Blog." which is what you'd get with `$title->getPrefixedText()`.